pride_and_prejudice.txt

```diff
@@ -1,6 +1,6 @@
 # Pride and Prejudice
 
-My _noun_  will not be repressed. You must allow me to _verb_ you how
+My _Horse__  will not be repressed. You must allow me to _JUMP_ you how
```

Comment:
Hey @TheRealAWebb, looks like we have some extra underscores here. We can remove these.